compaction/compaction_manager.hh

```diff
@@ -91,7 +91,19 @@ public:
                         // counted in compaction_manager::stats::errors
         };
         static std::string_view to_string(state);
-
+        struct compaction_task_stats {
```

Comment:
Do we still want to keep that struct (for documentation/future extension) or just single uint64_t?

Reply:
I think so.  It does't have any performance penalty and it makes the usage clearer.
Plus, it will help if and when we'd want to extend it.